frontend/src/components/widgets/space-wrapper/SpaceWrapper.jsx

```diff
@@ -1,16 +1,17 @@
 import { Space } from "antd";
 import PropTypes from "prop-types";
 
-const SpaceWrapper = ({ children }) => {
+const SpaceWrapper = ({ children, direction = "vertical" }) => {
   return (
-    <Space direction="vertical" className="width-100">
+    <Space direction={direction} className="width-100">
```

Comment:
@chandrasekharan-zipstack This component was implemented specifically for the vertical direction Space component, which requires 100% width. If you're looking to render items horizontally, you can simply use the <Space> component directly, as its default direction is horizontal. 

Can you please revert your changes in this component?

For reference: https://ant.design/components/space

Reply:
@tahierhussain This is done.